src/annotator/guest.js

```diff
@@ -67,11 +67,9 @@ const IGNORE_SELECTOR = '[class^="annotator-"],[class^="hypothesis-"]';
 
 export default class Guest extends Delegator {
   constructor(element, config, anchoring = htmlAnchoring) {
-    // TODO - Find out if `defaultConfig` actually does anything and remove it
-    // if not.
     const defaultConfig = {
+      // This causes the `Document` plugin to be initialized.
       Document: {},
-      TextSelection: {},
```

Comment:
The empty objects here cause various "plugin" modules to be initialized. There is no `TextSelection` plugin any more however.